multi-casskop/helm/multi-casskop/templates/role.yaml

```diff
@@ -12,7 +12,7 @@ rules:
 - apiGroups:
   - db.orange.com
   resources:
-    - "cassandraclusters"
+    - "multicasskops"
```

Comment:
would it need access to the `cassandraclusters` resource as well?  it looks like multicasskop's reconciliation loop acts on those objects if i understand correctly

Reply:
@mattniedelman I think you're right. MultiCasskop As I remember :) needs to manipulate also cassandraclusters objects